src/modules/Providers/utils/validators/provider/vsphere/validateVCenterURL.ts

```diff
@@ -1,8 +1,18 @@
-import type { ValidationMsg } from 'src/providers/utils/types';
+import { type ValidationMsg, ValidationState } from '@utils/validation/Validation';
 
 import { safeBase64Decode } from '../../../helpers/safeBase64Decode';
 import { validateIpv4, validateURL } from '../../common';
 
+const getUrlObject = (url: string) => {
+  try {
+    return new URL(url);
+  } catch {
+    // eslint-disable-next-line no-console
+    console.error('Unable to parse URL.');
```

Comment:
We dont have MTVConsole or something similar? 
export const MTVConcolse = console;
Using MTVConsole, will not need this comment line